modules/product/src/Form/AddToCartForm.php

```diff
@@ -91,29 +107,20 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state, $product = NULL, array $settings = NULL) {
-    $variations = [];
-    foreach ($product->variations->referencedEntities() as $variation) {
-      $variations[$variation->id()] = $variation->label();
-    }
-
     $form['#settings'] = $settings;
 
+    $form_state->set('default_variation', $product->variations->first()->entity);
```

Comment:
Can this store the variation id as well? We want to keep the form and form state as slim as possible.